### ISSUE TITLE
[A11Y] Editorial View - Images/Alt text: Logo and linked images lack descriptive alternative text

### ISSUE THREAD

**Issue by israelcefrin** [pkp/pkp-lib#12596]
## Issue Description
Logo, journal thumbnail and HomePage settings allow Journal Editor to upload images without ALT text or leaving the Alternate text field empty. It results on images missing alternative text description for any image in any language in the frontend. Functional images without accessible names prevent screen reader users from understanding or activating core links.

## How to reproduce / Current behavior
1. Log in to OJS/OMP/OPS
2.  Go to Settings > website > Appearance > Setup
3. Upload a new Logo image
4. Leave Alternate text field empty
5. Click on the Save button
6. Go to the frontend homepage
7. Check the logo missing alt text

## What application are you using?
 OJS, OMP or OPS version 3.5.0-3

## Stack used 
- Device: Laptop
- OS: MacOS
- Browser: Firefox
- Version: Latest stable

## Testing method
- Method: Manual; visual inspection and measurement

## Action
Ensure all functional images expose an appropriate accessible name and decorative images are correctly ignored by assistive technologies. The backend **must** require image description to allow uploading images.

## Additional information
Image showing blank Alternate text field in OJS. backend, Website settings section.

<img width="1044" height="511" alt="Image showing blank Alternate text field in OJS. backend, Website settings section." src="https://github.com/user-attachments/assets/098d3fd1-ceba-40de-9796-e8de1fd534a0" />
